lib/libthr/thread/thr_umtx.c

```diff
@@ -30,15 +30,13 @@
 #include "thr_private.h"
 #include "thr_umtx.h"
 
-#ifndef HAS__UMTX_OP_ERR
-int _umtx_op_err(void *obj, int op, u_long val, void *uaddr, void *uaddr2)
+int __noinline
+_umtx_op_err(void *obj, int op, u_long val, void *uaddr, void *uaddr2)
```

Comment:
I suggest to move the fake implementation for other arches to libsys as well.  Then make the symbol MI.

Reply:
Do you think I should filter the symbol as done now or require that libthr link libsys for it? If we're going to filter then I think my current approach is cleanest as we have to have the symbol in libthr in order to filter it.

Reply:
IMO it is better to link libthr to libsys for _ump_op_err, and put the symbol in the private namespace.  Only libthr is the consumer of the symbol, not libc, and we do not want to export it for third-parties.